KlipTok.pl.json

```diff
@@ -15,12 +15,49 @@
 
 	// Login View
 	"DarkMode": "Ciemny motyw",
+	"LoginWithTwitch": "Zaloguj się z Twitch",
+	"Logout": "Wyloguj się",
 
+	// Header
+	"SearchForChannel": "Znajdź kanał...",
+	"SearchNotFoundFormat": "Nie znaleziono {0}",
 
 	// Sidebar
-	"SuggestedStreamers": "Proponowani streamerzy",
-	"FollowedStreamers": "Obserwowani streamerzy",
-	"AllStreamers": "Wszyscy streamerzy",
+	"SuggestedStreamers": "Polecane Kanały",
+	"FollowedStreamers": "Obserwowane Kanały",
+	"AllStreamers": "Wszystkie Kanały",
 	"About": "O nas",
-	"Close": "Zamknij"
+	"Close": "Zamknij",
+
+	// Streamer Page
+	"ClickToFollow": "Obserwuj",
+	"ClickToUnfollow": "Przestań Obserwować",
```

Comment:
```suggestion
	"ClickToUnfollow": "Przestań obserwować",
```

Reply:
If you look at en-US, that serves as a template, you will notice that in some cases words are using Upper case.
This is because those are labels of buttons, or some emphasized action.
So you have "Click to Unfollow" instead "click to unfollow"

Therefore your change it makes no sense in that context.

Reply:
In polish, we don't use upper case that way. And I don't get it when it makes no sense?

There are multiple places on Twitch when the upper case is used only in English.

Reply:
> In polish, we don't use upper case that way.

I admit that I do not have degree in Polish Philology, so will not dispute this.
But without knowing your background I take this statement lightly.

Definitelly you use capitalization in many areas of visual design, whether it is academically correct or not.

Reply:
> Definitelly you use capitalization in many areas of visual design, whether it is academically correct or not.

Show me a good example on Twitch where that happens.
In polish, upper case is used when you mean names of stuff e.g. Twitch.
When we choose visual design over polish rules then we have nothing to discuss.

Reply:
> When we choose visual design over polish rules then we have nothing to discuss.

As it happens, capitalization is very often used for something like button text values, even in Polish.
Whether it is gramatically correct in any language, it is different matter.
So yes, quite often such decisions are taken because of design and UX, not because gramathical rules of language.

Discussion is always good thing.